generators/README.md

```diff
@@ -4,8 +4,25 @@ This project contains experimental code to create a tool for generating Tekton s
 
 See [tektoncd/pipeline/#2590](https://github.com/tektoncd/pipeline/issues/2590) information and background.
 
+## GitHub Token
```

Comment:
This is secret is actually the webhook secret - https://developer.github.com/webhooks/securing/

The secret described here would be the one for fetching source code / updating PRs.